arithmetization/src/main/java/net/consensys/linea/zktracer/module/shakiradata/ShakiraData.java

```diff
@@ -76,8 +78,7 @@ public void call(final ShakiraDataOperation operation) {
 
     switch (operation.precompileType()) {
       case SHA256 -> sha256Blocks.addPrecompileLimit(operation.inputSize());
-      case KECCAK -> { // TODO
-      }
+      case KECCAK -> keccak.addPrecompileLimit(operation.inputSize());
```

Comment:
KECCAK isn't a precompile. I guess I'm confused why the KECCAK case would fall under a switch case like so
```java
  switch (operation.precompileType()) {
    case ...
}
```

Reply:
true, it's not a precompile at all. We should rename `precompileType` to `shakiraType` ?